### DISCUSSION TITLE
#7747: structuredClone()

### DISCUSSION THREAD

**Discussion by pjmattingly** [serverless/serverless#11229] [General]
Hi,

I was working on replacing _.merge in [7747](https://github.com/serverless/serverless/issues/7747), and I found that it may be possible to use [structuredClone()](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone). This requires a minimum node version, so I was trying to find out what the minimum node version supported by serverless is. I think it might be 14? Can someone clarify?

Thanks.

**Comment by mnapoli** [comment]
Hi, we still support Node 12 at the moment unfortunately.

**Comment by pjmattingly** [comment]
Ah! So close. Thanks. =)
